pg_search/src/postgres/build.rs

```diff
@@ -51,7 +51,7 @@ impl BuildState {
 
         BuildState {
             count: 0,
-            memctx: PgMemoryContexts::new("pg_search_index_build"),
+            per_row_context: PgMemoryContexts::new("pg_search ambuild context"),
```

Comment:
simply renamed this variable in `BuildState` to be consistent with the new one this PR adds to `InsertState`.